executor/ddl.go

```diff
@@ -821,7 +821,7 @@ func (e *DDLExec) getRecoverTableByTableName(tableName *ast.TableName) (*model.J
 		return nil, nil, err
 	}
 	if tableInfo == nil || jobInfo == nil {
-		return nil, nil, errors.Errorf("Can't find dropped/truncated table: %v in DDL history jobs", tableName.Name)
+		return nil, nil, errors.Errorf("Can't find dropped/truncated table: %v in DDL history jobs or it is a local temporary table", tableName.Name)
```

Comment:
I think keep the message with previous code is enough. But it also seems reasonable for current modification.